docs/samples/v1beta1/torchserve/v2/README.md

```diff
@@ -0,0 +1,108 @@
+# Predict on a InferenceService using TorchServe
```

Comment:
Is the correct folder to put all the V2 related samples? The folder name is xxx/v1beta1/xxx